cosmo_tester/test_suites/manager_of_managers/constants.py

```diff
@@ -15,32 +15,57 @@
 
 from cosmo_tester.framework.test_hosts import (
     REMOTE_OPENSTACK_CONFIG_PATH,
-    REMOTE_PRIVATE_KEY_PATH
+    REMOTE_PRIVATE_KEY_PATH,
+    REMOTE_PUBLIC_KEY_PATH
 )
 
-MOM_PLUGIN_VERSION = '2.0.1'
-MOM_PLUGIN_WGN_URL = 'https://github.com/Cloudify-PS/manager-of-managers/releases/download/v{0}/cloudify_manager_of_managers-{0}-py27-none-linux_x86_64.wgn'.format(MOM_PLUGIN_VERSION)  # NOQA
-MOM_PLUGIN_YAML_URL = 'https://github.com/Cloudify-PS/manager-of-managers/releases/download/v{0}/cmom_plugin.yaml'.format(MOM_PLUGIN_VERSION)  # NOQA
+MOM_PLUGIN_REPO_PATH = 'cloudify-cosmo/cloudify-spire-plugin'
+MOM_PLUGIN_VERSION = '3.0.0'
```

Comment:
Maybe try to get the latest release? Or we want to update the version right before the release

Reply:
In the past we kept this static.

Reply:
The question if we will remember to update this

Reply:
This is in keeping with the existing practice of MOM.

Reply:
I think what I prefer to do, is to have another project for generalizing plugin use system tests to address this.